guides/source/action_cable_overview.md

```diff
@@ -542,7 +542,7 @@ test:
 
 production:
   adapter: redis
-  url: redis://10.10.3.153:6381
+  url: redis://user:secret@localhost:6381/12?foo=bar&qux=baz
```

Comment:
Ah yes, you're right. While a multi-user feature is in the works, we're not quite there yet.